cypress.config.js

```diff
@@ -2,7 +2,7 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
-    baseUrl: 'https://www.demoblaze.com/',
+    baseUrl: 'https://www.demoblaze.com',
```

Comment:
The base URL has been changed from 'https://www.demoblaze.com/' to 'https://www.demoblaze.com'. It is recommended to include the trailing slash at the end of the URL to avoid potential relative path issues.